public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+/*---general page and components css---*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+* {
+  overflow: auto;
+}
+
+body {
+  width: 100%;
+  margin: auto;
+  font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-weight: 300;
+}
+
+nav {
+  margin-left: -1em;
+}
+nav li {
+  display: inline-block;
+}
+
+nav, section {
+  text-align: center;
+ }
+header > p {
+    margin-bottom: -.7em;
+  }
+section {
+  width: 100%;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 2em;
+}
+
+ul {
+    margin-left: -1em;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1 {
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  margin-bottom: 2em;
+  margin-top: 2em;
+}
+
+/* -- lines created on h1 -- */
+
+h1:after, h1:before {
+  top: 51%;
+  width: 32%;
+  content: "";
+  position: absolute;
+  overflow: hidden;
+  height: 1px;
+  background-color: #585050;
+  margin-left: 3%;
+}
+
+h1:before {
+  margin-left: -35%;
+  text-align: right;
+}
+
+header {
+  margin-top: 3em;
+  margin-bottom: -3em;
+}
+
+.section-orange > h1:before, .section-orange > h1:after {
+  background-color: #fff;
+}
+
+.section-black > h1:before, .section-black > h1:after {
+  background-color: #fff;
+}
+
+.clip {
+  margin-left: 3em;
+}
+
+.subheader {
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  margin-bottom: 2em;
+  margin-top: 2em;
+}
+.subheader:before, .subheader:after {
+  top: 51%;
+  width: 20%;
+  content: "";
+  position: absolute;
+  overflow: hidden;
+  height: 1px;
+  background-color: #585050;
+  margin-left: 2%;
+}
+.subheader-skills:before, .subheader-skills:after {
+  background-color: #fff;
+}
+.subheader:before {
+  margin-left: -22%;
+  text-align: right;
+}
+.section-orange > .subheader:before, .section-orange > .subheader:after {
+  background-color: #fff;
+}
+.section-white > .subheader:before, .section-white > .subheader:after {
+  background-color: #000;
+}
+
+.column-two {
+  -webkit-column-count: 2;
+     -moz-column-count: 2;
+     -ms-column-count: 2;
+          column-count: 2;
+}
+.column-right {
+  float: right;
+}
+.section-orange {
+  background-color: #FF6124;
+  color: #fff;
+  padding-bottom: 2em;
+}
+
+.section-black {
+  background-color: #585050;
+  color: #fff;
+}
+
+/*---Navigation menu css---*/
```

Comment:
Aesthetically, this navigation menu doesn't look like it goes with the original design. It's not using much of the design "language" as the rest of the page.